jobs/webcompat-kb/webcompat_kb/main.py

```diff
@@ -78,283 +100,543 @@ def get_urls_from_story(user_story_str):
         "source": "see_also",
         "condition": STANDARDS_POSITIONS,
     },
-    "url_patterns": {
-        "fields": [
-            {"name": "bug", "type": "INTEGER", "mode": "REQUIRED"},
-            {"name": "url_pattern", "type": "STRING", "mode": "REQUIRED"},
-        ],
-        "source": "cf_user_story",
-        "custom_func": get_urls_from_story,
-    },
 }
 
 LINK_FIELDS = ["other_browser_issues", "standards_issues", "standards_positions"]
 
 
-def fetch_bugs(params=None):
-    if params is None:
-        params = {}
-
-    fields = [
-        "id",
-        "summary",
-        "status",
-        "resolution",
-        "product",
-        "component",
-        "see_also",
-        "depends_on",
-        "blocks",
-        "priority",
-        "severity",
-        "cf_user_story",
-    ]
+def extract_int_from_field(field):
+    if field:
+        if field.lower() in FIELD_MAP:
+            return FIELD_MAP[field.lower()]
 
-    url = f"{BUGZILLA_API}/bug"
-    params["include_fields"] = ",".join(fields)
+        match = re.search(r"\d+", field)
+        if match:
+            return int(match.group())
 
-    response = requests.get(url, params=params)
-    response.raise_for_status()
-    result = response.json()
+    return None
 
-    return result["bugs"]
 
+def parse_string_to_json(input_string):
+    if not input_string:
+        return ""
+
+    input_string = input_string.replace("\r\n", "\n")
+    lines = input_string.split("\n")
```

Comment:
I think `.splitlines()` would work here, but not a blocker.

Reply:
good point, thanks! I've changed it to use `.splitlines()`